src/kvirc/ui/KviInputEditor.cpp

```diff
@@ -63,16 +63,11 @@
 #include <QFontMetrics>
 #include <QKeyEvent>
 #include <QDragEnterEvent>
-#if (QT_VERSION < 0x050000)
-	#include <QInputContext>
```

Comment:
why not add #include <QInputContext>? and also what about the https://github.com/wodim/KVIrc/blob/qt5/src/kvirc/ui/KviInputEditor.cpp#L1136-L1152